afs-storage-api/src/main/java/com/powsybl/afs/storage/check/FileSystemCheckIssue.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage.check;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class FileSystemCheckIssue {
+
+    private UUID uuid;
+    private String name;
+    private Type type;
+    private boolean repaired;
```

Comment:
Could you add a `description` and a `resolutionDescription` field to provide more information about the error and its resolution?

For example it could be, for inconsistent nodes:
description : "Node \<ID\> is inconsistent and older than \<duration\>."
resolutionDescription: "Node \<ID\> has been deleted."